R/methods_marginaleffects.R

```diff
@@ -55,21 +58,29 @@ model_parameters.marginalmeans <- model_parameters.marginaleffects
 
 #' @rdname model_parameters.averaging
 #' @export
-model_parameters.deltamethod <- model_parameters.marginaleffects
+model_parameters.hypotheses <- model_parameters.marginaleffects
+
+
+#' @rdname model_parameters.averaging
+#' @export
+model_parameters.slopes <- model_parameters.marginaleffects
 
 
 #' @rdname model_parameters.averaging
 #' @export
 model_parameters.predictions <- function(model,
                                          ci = 0.95,
                                          ...) {
-  insight::check_if_installed("marginaleffects")
-  out <- insight::standardize_names(model, style = "easystats")
+  insight::check_if_installed("marginaleffects", minimum_version = "0.9.0")
+
+  out <- datawizard::data_rename(model, "estimate", "predicted") |> data.frame()
+  out <- datawizard::data_relocate(out, "predicted", before = 1)
```

Comment:
No (native) pipe in package code, please ;-)

Reply:
hahaha